planetiler-custommap/src/main/resources/samples/shortbread.yml

```diff
@@ -149,6 +149,54 @@ layers:
     - *water_lines_attr_tunnel
     - *water_lines_attr_bridge
 
+- id: dam_lines
+  features:
+  - source: osm
+    geometry: line
+    min_zoom: 12
+    include_when: &dam_filter
+      # note that dykes are not dams
+      waterway: dam
+    attributes: &dam_atrs
+    - key: kind
+      value: dam
+
+- id: dam_polygons
+  features:
+  - source: osm
+    geometry: polygon
+    min_zoom: 12
+    include_when: *dam_filter
+    attributes: *dam_atrs
+
+- id: pier_lines
+  features:
+  - source: osm
+    geometry: line
+    min_zoom: 12
+    include_when:
+      man_made: 
+        - pier
+        - breakwater
+        - groyne
+    attributes:
+    - key: kind
+      type: match_value
+
+- id: pier_polygons
+  features:
+  - source: osm
+    geometry: polygon
+    min_zoom: 12
+    include_when:
+      man_made: 
+        - pier
+        - breakwater
+        - groyne
+    attributes:
+    - key: kind
+      type: match_value
```

Comment:
not done as below because of the 50 alias limit:
![image](https://github.com/user-attachments/assets/f22df0e0-8642-49b4-90e8-d3e0b38b872f)


```suggestion
    include_when: &pier_filter
      man_made: 
        - pier
        - breakwater
        - groyne
    attributes: &pier_attrs
    - key: kind
      type: match_value

- id: pier_polygons
  features:
  - source: osm
    geometry: polygon
    min_zoom: 12
    include_when: *pier_filter
    attributes: *pier_attrs
```

Reply:
Yes, if https://github.com/onthegomap/planetiler/pull/1224 is merged, this (and other similar comments) can be committed without breaking the verification ^^